framework/pages/footer.ts

```diff
@@ -0,0 +1,64 @@
+import { type Page, expect } from "@playwright/test";
+
+const url = process.env.BASE_URL + process.env.BASE_LOCALIZATION + process.env.CONTROL_REBRANDED_TEST_VARIANT;
+
+const jobs = '//*[text()="Jobs"]';
+const social = '//*[text()="Social"]';
+const app = '//div[text()="Social"]/..//following-sibling::div/div[text()="Get the App"]';
+const socialElements = "//*[text()='Social']/..//li/a";
+const appElements = "//div[text()='Social']/..//following-sibling::div/div[text()='Get the App']/..//li/a";
+const footerLink = "//footer//a[@href='https://www.studeersnel.nl']";
```

Comment:
could you please clean up locators? They should look much simpler. Also, we can add in `utils.ts` every repetitive job to work with locators. For example, getting an attribute:

`const getAttribute = async function (page: Page, dataTestSelector: string, attributeName: string ) {
    return page.locator('[data-test-selector="' + dataTestSelector + '"]').getAttribute(attributeName);
}`


```suggestion
const jobs = 'jobs-link';
...

```
and similar to this template. You can take a look at other pages as example

Reply:
updated